projects/plugins/boost/app/modules/critical-css/class-critical-css.php

```diff
@@ -763,9 +763,19 @@ public function force_proxied_css( $src ) {
 
 		$parsed = wp_parse_url( $src );
 
-		// If no domain specified, or domain matches current, no need to proxy.
+		// Build the resource origin host the requested asset belongs to.
+		$resource_origin = '';
+		if ( isset( $parsed['port'] ) ) {
```

Comment:
I suspect this `if` statement shouldn't be here. Shouldn't `$resource_origin` always start with $parsed['host']? Otherwise, `$resource_origin` will be empty for any URL without a port specified.

Reply:
That is a silly mistake 🤦🏽‍♂️